server/bot/commands/utilities.ts

```diff
@@ -518,3 +518,48 @@ export const handleEditLastSay: Command = {
     } catch (e) { return 'Nothing to edit.' }
   }
 }
+
+export const handleSuppress: Command = {
+  name: 'suppress',
+  opts: {
+    requirements: {
+      permissions: { manageMessages: true }
+    },
+    description: 'Suppress or unsuppress embeds in a message.',
+    fullDescription: 'Suppress embeds in a message. If the message already has a suppressed embed, it will get unsuppressed. Requires \'Manage Messages\' permission',
+    usage: '/suppress (channel) <message ID or link>',
+    example: '/suppress #general 123456789012345678'
+  },
+  generator: async (message, args, { client }) => {
+    let msg
+    let channelToEdit
+    if (args.length === 1) {
+      const regex = /https?:\/\/((canary|ptb|www).)?discord(app)?.com\/channels\/\d{17,18}\/\d{17,18}\/\d{17,18}/
+      if (regex.test(args[0])) {
+        const split = args[0].split('/')
+        const channel = message.member.guild.channels.get(split[5]) as GuildTextableChannel
+        msg = channel.messages.get(split[6]) || await client.getMessage(split[5], split[6])
+        channelToEdit = split[5]
```

Comment:
Store channels instead of ids.
```suggestion
        channel = message.member.guild.channels.get(split[5]) as GuildTextableChannel
        if (!channel) return `That's not a real channel, you ${getInsult()}.`
        msg = channel.messages.get(split[6]) || await channel.getMessage(split[6]
```